docs/source/baselines.rst

```diff
@@ -72,3 +84,164 @@ Launch training
+While DEPRL can be used for any kind of RL task, we provide a pre-trained controller and training settings for the `myoLegWalk-v0` task.
+See `this tutorial <https://github.com/P-Schumacher/myosuite/blob/main/docs/source/tutorials/4a_deprl.ipynb>`_ for more detailed tutorials.
+
+Installation
+~~~~~~~~~~~~
+Simply run
+
+.. code-block:: bash
+
+    python -m pip install deprl
+
+after installing the myosuite.
+
+Train new policy
+~~~~~~~~~~~~~~~~
+
+For training from scratch, navigate to the `agents folder <https://github.com/facebookresearch/myosuite/tree/main/myosuite/agents/>`_ folder and run the following code
+
+.. code-block:: bash
+
+    python -m deprl.main baselines_DEPRL/myoLegWalk.json
+
+and training should start. Inside the json-file, you can set a custom output folder with ``working_dir=myfolder``. Be sure to adapt the ``sequential`` and ``parallel`` settings. During a training run, `sequential x parallel` environments are spawned, which consumes over 30 GB of RAM with the default settings for the myoLeg. Reduce this number if your workstation has less memory.
+
+Visualize, log and plot
+~~~~~~~~~~~~~~~~~~~~~~~
+We provide several utilities in the ``deprl`` package.
+To visualize a trained policy, run
+
+.. code-block:: bash
+
+    python -m deprl.play --path baselines_DEPRL/myoLegWalk_20230514/myoLeg/
+
+
+where the last folder contains the ``checkpoints`` and ``config.yaml`` files. This runs the policy for several episodes and returns scores.
+You can also log some settings to `wandb <https://wandb.ai/)>`_ . Set it up and afterwards run
+
+.. code-block:: bash
+
+    python -m deprl.log --path baselines_DEPRL/myoLegWalk_20230514/myoLeg/log.csv --project myoleg_deprl_baseline
+
+which will log all the training metrics to your ``wandb`` project.
+
+If you want to plot your training run, use
+
+.. code-block:: bash
+
+    python -m deprl.plot --path baselines_DEPRL/myoLegWalk_20230514/
+
+
+For more instructions on how to use the plot feature, checkout `TonicRL <https://github.com/fabiopardo/tonic>`_, which is the general-purpose RL library deprl was built on.
+
+
+
+.. _reflex_controller:
+
+MyoLegReflex baseline
+`````````````````````
+
+MyoLegReflex is a reflex-based walking controller for MyoLeg. With the provided set of 46 control parameters, MyoLeg generates steady walking patterns. Users have the freedom to discover alternative parameter sets for generating diverse walking behaviors or design a higher-level controller that modulates these parameters dynamically, thereby enabling navigation within dynamic environments.
+
+Examples
+~~~~~~~~~~~~~~
+
+MyoLegReflex is bundled as a wrapper around MyoLeg. To run MyoLegReflex with default parameters, you can either utilize the Jupyter notebook found at ``myosuite/agents/baseline_Reflex`` or execute the following code snippet:
```

Comment:
@elladyr you might want to indicate that this works only from the spcific folder so it is not accessible from the general myosuite api

Reply:
I have indicated that this only works from the folder in my own commit to the v152 branch